src/SDKs/Azure.ApplicationModel.Configuration/data-plane/README.md

```diff
@@ -1,25 +1,62 @@
+
+To create a Configuration Store, you can use the Azure Portal or [Azure CLI][azure_cli] with the following command:
+
+```Powershell
+az appconfig create --name <config-store-name> --resource-group <resource-group-name> --location eastus
+```
+
+### Authenticate the client
+
+In order to interact with the App Configuration service, you'll need to create an instance of the [Configuration Client][configuration_client_class] class. To make this possible, you'll need the connection string of the Configuration Store.
+
+#### Get credentials
+Use the [Azure CLI][azure_cli] snippet below to get the connection string from the Configuration Store.
+```Powershell
+az appconfig credential list --name <config-store-name>
+```
+
+Alternatively, get the connection string from the Azure Portal.
+
+#### Create client
+
+Once you have the value of the connection string, you can create the ConfigurationClient:
+
+```c#
+string connectionString = <connection_string>;
+var client = new ConfigurationClient(connectionString);
+```
+
+## Key concepts
+
+### Configuration Setting
 Is the fundamental resource within a Configuration Store. In its simplest form it is a key and a value. However, there are additional properties such as the modifiable content type and tags fields that allow the value to be interpreted or associated in different ways.
 
-The Label property of a configuration setting provides a way to separate configuration settings into different dimensions. These dimensions are user defined and can take any form. Some common examples of dimensions to use for a label include regions, semantic versions, or environments. Many applications have a required set of configuration keys that have varying values as the application exists across different dimensions.
-For example, MaxRequests may be 100 in "NorthAmerica", and 200 in "WestEurope". By creating a configuration setting named MaxRequests with a label of "NorthAmerica" and another, only with a different value, in the "WestEurope" label, a solution can be achieved that allows the application to seamlessly retrieve configuration settings as it runs in these two dimensions.
+The Label property of a Configuration Setting provides a way to separate Configuration Settings into different dimensions. These dimensions are user defined and can take any form. Some common examples of dimensions to use for a label include regions, semantic versions, or environments. Many applications have a required set of configuration keys that have varying values as the application exists across different dimensions.
+For example, MaxRequests may be 100 in "NorthAmerica", and 200 in "WestEurope". By creating a Configuration Setting named MaxRequests with a label of "NorthAmerica" and another, only with a different value, in the "WestEurope" label, a solution can be achieved that allows the application to seamlessly retrieve Configuration Settings as it runs in these two dimensions.
```

Comment:
"a solution can be achieved that allows the application to seamlessly retrieve" is wordy. Could consider replacing with "an application can seamlessly retrieve".